app/views/departureP5/IntentionToControlP5View.scala.html

```diff
@@ -31,37 +30,38 @@
 config: FrontendAppConfig
 )
 
-@(intentionToControlP5ViewModel: IntentionToControlP5ViewModel, departureId: String, customsOfficeContactViewModel: CustomsOfficeContactViewModel)(implicit request: Request[?], messages: Messages)
+@(intentionToControlP5ViewModel: IntentionToControlP5ViewModel, departureId: String, messageId: String, customsOfficeContactViewModel: CustomsOfficeContactViewModel)(implicit request: Request[?], messages: Messages)
 
     @mainTemplate(
         title = intentionToControlP5ViewModel.titleAndHeading,
         showBackLink = true
     ) {
 
-    @heading(
-        heading = intentionToControlP5ViewModel.titleAndHeading,
-        caption = intentionToControlP5ViewModel.lrn.map(x => messages("departure.messages.caption", x))
-    )
+    @formHelper(action = controllers.departureP5.routes.IntentionToControlP5Controller.onSubmit(departureId, messageId), Symbol("autoComplete") -> "off") {
```

Comment:
Again very minor but if we're wanting to sort out the indentation here it looks like the mainTemplate block can be un-indented 😄